index.js

```diff
@@ -0,0 +1,108 @@
+import {
+  lcontact, contact21, section31, list11,
+} from './modules/contact.js';
+import {
+  add, contact2, section21, list1,
+} from './modules/add.js';
+import {
+  list, lList, contact, section2,
+} from './modules/list.js';
+import setInitialTime from './modules/dateTime.js';
+
+class Awesomebooks {
+  constructor() {
+    this.data = JSON.parse(localStorage.getItem('data')) || [];
+  }
+
+  add = () => {
+    const author = document.querySelector('.author');
+    const title = document.querySelector('.title');
+    const error = document.querySelector('.error-message');
+
+    if (author.value === '' || title.value === '') {
+      const errormessage = new Error('please privide title and author');
+      error.innerHTML = errormessage;
+      return;
+    }
+    const obj = { author: author.value, title: title.value };
+    this.data.push(obj);
+    this.updateLocalStorage(this.data);
+    author.value = '';
+    title.value = '';
+    this.displayItems();
+  };
+
+  updateLocalStorage = () => {
+    localStorage.setItem('data', JSON.stringify(this.data));
+  };
+
+  displayItems = () => {
```

Comment:
- [ ] @enocol Great work implementing the change requested, however, the list of books disappears from the homepage when I refresh the page, for me to view the books I will need to add a new book.

Reply:
@danmainah fixed now. Thanks